test/export/ExtensionExporter.test.ts

```diff
@@ -46,8 +46,75 @@ describe('ExtensionExporter', () => {
     const extensionBar = new Extension('Bar');
     doc.extensions.set(extensionFoo.name, extensionFoo);
     doc.extensions.set(extensionBar.name, extensionBar);
-    const exported = exporter.export(input);
+    const exported = exporter.export();
     expect(exported.length).toBe(1);
     expect(exported[0].name).toBe('Bar');
   });
+
+  it('should export extensions with FSHy parents', () => {
```

Comment:
I am willing to be wrong on this because I'd rather have more tests than less, but couldn't you make a case that the new tests here for `ExtensionsExporter` and `ProfileExporter` are testing the `exportStructDef` function in the same way? I think it might make more sense to have maybe one test in each of these files that ensures basic FSHy paths works for both Extensions and Profiles, but to then have one set of more in depth tests in the `StructureDefinitionExporter.test.ts` file.

Reply:
I'm not sure I agree in this specific case. It seems to me that it would be better to test individually in each exporter because technically parents should only work from profile to profile or extension to extension. Adding tests to the base `StructureDefinitionExporter` would require either picking one or doing both. I suppose that's also fine, but I think I personally prefer it this way. Especially given the fact that the rest of the `ProfileExporter` and `ExtensionExporter` tests are duplicative in this way.

Reply:
That seems fair to me, I'd rather more tests than too few anyway.